skll/learner.py

```diff
@@ -1633,7 +1627,7 @@ def train(self, examples, param_grid=None, grid_search_folds=3,
                 else:
                     grid_jobs = min(num_specified_folds, grid_jobs)
                 # Only retain IDs within folds if they're in grid_search_folds
-                dummy_label = next(itervalues(grid_search_folds))
+                dummy_label = next(iter(grid_search_folds.values()))
```

Comment:
```suggestion
                dummy_label = next(grid_search_folds.values())
```

I think `values()` is already an iterator?

Reply:
No, that gave me an error, so had to use `iter`.

Reply:
Ah, right. Because `next()` only works on iterators.